Packages/io.chainsafe.web3-unity.hyperplay/Runtime/Scripts/HyperPlayWebGLProvider.cs

```diff
@@ -84,7 +86,7 @@ public override async Task<string> Connect()
         /// <param name="parameters">RPC request parameters.</param>
         /// <typeparam name="T">RPC request response type.</typeparam>
         /// <returns>RPC request Response.</returns>
-        public override async Task<T> Perform<T>(string method, params object[] parameters)
+        public override async Task<T> Request<T>(string method, params object[] parameters)
```

Comment:
Okay that name makes way more sense 😄

Reply:
yep, so `WalletProvider` is now an `IRpcProvider`, `IClient` and `IWalletProvider` which works great for all connection providers!